### DISCUSSION TITLE
Mark "Verify login" notification as Time Sensitive in iOS

### DISCUSSION THREAD

**Discussion by juanvisbal** [community/community#28967] [Mobile] [answered]
I have notifications set to Scheduled Summary but I would like the notification to verify a login request to be shown immediately.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.75.0 (74400986)
Device: iPhone 13 mini (iOS 16.0)
</details>

**Comment by alcere** [comment, marked as answer]
Hi! @juanvisbalco thanks for the feedback! I am sharing it with the team to get the idea evaluated! thanks again!
